### ISSUE TITLE
ENMtools not on CRAN

### ISSUE THREAD

**Issue by plantarum** [danlwarren/ENMTools#227]
Hi,

I can't install ENMTools from CRAN: 

```
Warning message:       
package ‘ENMTools’ is not available for this version of R
```

It isn't listed on the main [CRAN webpage](https://cran.r-project.org/web/packages/available_packages_by_name.html#available-packages-E) today (November 15, 2022). It is listed in the [archive](https://cran.r-project.org/src/contrib/Archive/ENMTools/), which (maybe?) indicates the last change was 12 November 2022?

![ENMTools_000](https://user-images.githubusercontent.com/5727138/202033311-a65b2ba3-834d-4679-bb8a-6be02a5e5136.jpg)

I don't see any notices about this in the mailing lists or in your issue tracker. Do you know why it's not showing up on CRAN?

Tyler

**Comment by danlwarren** [comment]
Well damn.  Thanks for pointing this out!  

There was an issue with ecospat that we got a warning for, with ENMTools being taken offline if we didn't remove that dependency.  Ecospat was updated so I assumed that would mean we wouldn't be taken offline, but I think the changes just went through too slowly.  I'll try to submit a new version today.  It'll still be a few days before it's on CRAN though, so you'll need to install from github usint devtools for now.
